src/components/book-list/book-list.html

```diff
@@ -0,0 +1,23 @@
+<div class="master-container list-container">
+  <div class="navbar-container">
+    <div class="navbar">
+      <img class="wolox-logo" src="../../assets/LogoWolox.png" alt="wolox logo">
+      <input class="book-search" ng-model="searchText" type="text" placeholder="Buscá por título del libro...">
+      <button class="logout">Logout</button>
+    </div>
+  </div>
+  <div class="master-book-list-container">
+    <div class="book-container" ng-repeat="book in $ctrl.bookList | filter:searchtxt">
+      <img class="cover-page-list" ng-src="{{book.image_url || 'assets/book-cover.png'}}"
+        alt="{{book.image_url ? 'Cover page' : 'Wolox books cover page'}}" />
```

Comment:
Just an indentation suggestion
```suggestion
<img
  class="cover-page-list"
  ng-src="{{book.image_url || 'assets/book-cover.png'}}"
  alt="{{book.image_url ? 'Cover page' : 'Wolox books cover page'}}"
/>
```